guide/onboarding/backing-up-a-recovery-phrase.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Backing up a recovery phrase
+description: Handling recovery phrases during onboarding
+parent: Onboarding
+has_children: true
+nav_order: 3
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/
+main_classes: -no-top-padding
+---
+
+# Backing up a recovery phrase
+
+When a user generates a new wallet, it creates something called a private key (also known as a recovery phrase). A [recovery phrase]({{ 'guide/glossary/#recovery-phrase' | relative_url }}), is a group of 12 or 24 words that let users recover funds if wallet access is lost. Likewise, users who lose both their recovery phrase and wallet access permanently lose their bitcoin. Arguably, this creates one of the toughest design problems within the space.
```

Comment:
> it creates something called a private key

I don't think recovery phrase is called private key, is it?

Reply:
🚧